library/ws-widget/collection/src/lib/_common/content-toc/confirmation-modal/confirmation-modal.component.html

```diff
@@ -1,26 +0,0 @@
-<div class="modal-container">
-    <!-- Enroll content -->
-    <ng-container *ngIf="!data.enroll">
-        <div class="flex flex-col gap-4 text-center">
-            <div class="ws-mat-default-text font-bold">You’re one step away from enrolling!</div>
-            <div class="text-sm">This batch starting on 12/06/2023 - 23/06/2023, kindly go through the content and be
-                prepared.</div>
-            <div class="flex flex-col sm:flex-row gap-4">
-                <button class="flex justify-center cancel-button" (click)="handleCloseModal()">Cancel</button>
```

Comment:
This PR should not have any code related to TOC Page, please check

Reply:
Those are extra codes, which we are not at all using. Hence I cleaned up the code.